ZuluContent/SkillHandlers/Stealth.cs

```diff
@@ -4,87 +4,89 @@
 namespace Server.SkillHandlers
 {
     public class Stealth
-	{
-		public static void Initialize()
-		{
-			SkillInfo.Table[(int)SkillName.Stealth].Callback = OnUse;
-		}
+    {
+        public static void Initialize()
+        {
+            SkillInfo.Table[(int) SkillName.Stealth].Callback = OnUse;
+        }
 
-		public static double HidingRequirement { get { return 80.0; } }
+        public static double HidingRequirement
+        {
+            get { return 80.0; }
+        }
 
-		public static int[,] ArmorTable { get { return m_ArmorTable; } }
-		private static int[,] m_ArmorTable = new[,]
-			{
-							//	Gorget	Gloves	Helmet	Arms	Legs	Chest	Shield
-				/* Cloth	*/	{ 0,	0,		0,		0,		0,		0,		0 },
-				/* Leather	*/	{ 0,	0,		0,		0,		0,		0,		0 },
-				/* Studded	*/	{ 2,	2,		0,		4,		6,		10,		0 },
-				/* Bone		*/ 	{ 0,	5,		10,		10,		15,		25,		0 },
-				/* Spined	*/	{ 0,	0,		0,		0,		0,		0,		0 },
-				/* Horned	*/	{ 0,	0,		0,		0,		0,		0,		0 },
-				/* Barbed	*/	{ 0,	0,		0,		0,		0,		0,		0 },
-				/* Ring		*/	{ 0,	5,		0,		10,		15,		25,		0 },
-				/* Chain	*/	{ 0,	0,		10,		0,		15,		25,		0 },
-				/* Plate	*/	{ 5,	5,		10,		10,		15,		25,		0 },
-				/* Dragon	*/	{ 0,	5,		10,		10,		15,		25,		0 }
-			};
+        public static int[,] ArmorTable
```

Comment:
[,] ?

Reply:
`[,]` is part of the syntax for multi-dimensional arrays

https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/arrays/multidimensional-arrays

Reply:
Ah cool!